packages/plugins/Draw/src/components/Draw.vue

```diff
@@ -16,6 +16,7 @@
         :values="selectableDrawModes"
         :change-callback="setDrawMode"
       ></RadioCard>
+      <DrawOptions v-if="showDrawOptions" />
```

Comment:
@warm-coolguy do you remember the reason why we discouraged self-closing components even though they are [strongly recommended](https://v2.vuejs.org/v2/style-guide/#Self-closing-components-strongly-recommended) by Vue in SFC?

Reply:
It was due to a [https://stackoverflow.com/questions/56572975/vue-doesnt-render-second-component-if-the-previous-self-terminates](parcel bug). Since we've switched since, there should be no issue with this anymore, and we should be able to safely remove all discouragements. (Maybe test it before doing it, though.)